modules/income_and_assets/lib/income_and_assets/pdf_fill/forms/va21p0969.rb

```diff
@@ -559,6 +565,52 @@ class Va21p0969 < FormBase
             question_suffix: '(m)',
             question_text: 'DO YOU HAVE ANY ADDITIONAL AUTHORITY OR CONTROL OF THE TRUST?'
           }
+        },
+        # 10a
+        'unreportedAsset' => { key: 'F[0].#subform[8].DependentsHaveAssetsNotReported10a[0]' },
+        'unreportedAssets' => {
+          limit: 1,
+          first_key: 'assetOwnerRelationship',
+          # 10b
+          'assetOwnerRelationship' => {
+            'veteran' => { key: "F[0].RelationshipToVeteran10[#{ITERATOR}]" },
+            'spouse' => { key: "F[0].RelationshipToVeteran10[#{ITERATOR}]" },
+            'custodianOfChild' => { key: "F[0].RelationshipToVeteran10[#{ITERATOR}]" },
+            'child' => { key: "F[0].RelationshipToVeteran10[#{ITERATOR}]" },
+            'parent' => { key: "F[0].RelationshipToVeteran10[#{ITERATOR}]" },
+            'other' => { key: "F[0].OtherRelationship10[#{ITERATOR}]" }
+          },
+          'assetOwnerRelationshipOverflow' => {
+            question_num: 10,
+            question_suffix: '(b)',
+            question_text: 'SPECIFY ASSET OWNER\'S RELATIONSHIP TO THE VETERAN'
+          },
+          # 10c
+          'assetType' => { key: 'F[0].Page_10[0].TypeOfAsset10[0]' },
```

Comment:
For this one you should be able to do this:
```ruby
'assetType' => {
  key: 'F[0].Page_10[0].TypeOfAsset10[0]' },
  question_num: 10,
  question_suffix: '(2)',
  question_text: 'SPECIFY TYPE OF ASSET (CASH, ART, ETC.)'
},
```
and get rid of the `assetTypeOverflow` key